src/components/List.js

```diff
@@ -15,77 +15,36 @@ class List extends React.Component {
     getSort() {
         switch (this.props.sort) {
             case "lastUsed":
-                return (a,b) => this.lastUsedSort(a,b);
+                return (a,b) => this.sortBy(a,b, "lastUsed", "name");
             case "numUses":
-                return (a,b) => this.mostUsedSort(a,b);
+                return (a,b) => this.sortBy(a,b, "uses", "name");
             case "group":
-                return (a,b) => this.groupSort(a,b);
+                return (a,b) => this.sortBy(a,b, "group", "name");
             case "plus/basic":
-                return (a,b) => this.plusBasicSort(a,b);
+                return (a,b) => this.sortBy(a,b, "category", "name");
             case "userPosition":
-                return (a,b) => this.userSort(a,b);
+                return (a,b) => this.sortBy(a,b, "position", "");
             default:
-                return (a,b) => this.alphabeticalSort(a,b);
+                return (a,b) => this.sortBy(a,b, "name", "");
         }
     }
 
-    alphabeticalSort(a, b) {
-        if (a.name < b.name) {
-            return -1;
-        } else if (a.name > b.name) {
-            return 1;
-        } else {
-            return 0;
-        }
-    }
-
-    groupSort(a, b) {
-        if (a.group < b.group) {
-            return 1;
-        } else if (a.group > b.group) {
-            return -1;
-        } else {
-            return this.alphabeticalSort(a,b);
-        }
-    }
-
-    plusBasicSort(a, b) {
-        if (a.category === b.category) {
-            return this.alphabeticalSort(a,b);
-        } else if (a.category === "plus") {
-            return -1;
-        } else {
-            return 1;
+    sortBy(a,b, attribute, subAttribute) {
+        var reverse = false;
```

Comment:
"Reverse" implies that it is reversed sort. A before B is natural sort order, and the others are technically reverse sort order. I would probably make the variable be "ascending" as then you wouldn't need to make any other logic changes and it would make more sense to anyone else reviewing later.